sigopt/interface.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, requestor):
       endpoints=[
         ApiEndpoint(None, Experiment, 'POST', 'create'),
         ApiEndpoint(None, object_or_paginated_objects(Experiment), 'GET', 'fetch'),
+        ApiEndpoint(None, object_or_paginated_objects(Experiment), 'GET', 'progress'),
```

Comment:
this should probably be an `AggregatedObservation` (or whatever name we want to give this)